src/pages/signup.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link } from 'gatsby';
+import * as React from 'react';
+
+import styled from '@styled-components';
+import { Layout, Seo } from '@components/shared';
+import CtaButton from '@components/index-page/cta-button';
+import { useSiteMetadata } from '@hooks';
+import Form, {
+  FormLabel,
+  FormInput,
+  LinkWrapper,
+  ButtonWrapper,
+} from '@components/shared/form';
+import { useState } from 'react';
+
+const Wrapper = styled.section`
+  background-color: ${({ theme }) => theme.colors.section};
+
+  padding: ${({ theme }) => theme.boxes.padding.section.smallTop};
+
+  @media screen and(max-width: ${({ theme }) => theme.sizes.width.medium}) {
+    flex-direction: column;
+  }
+
+  @media screen and(max-width: ${({ theme }) => theme.sizes.width.small}) {
+    padding: ${({ theme }) => theme.boxes.padding.section.small};
+  }
+`;
+
+const SignUpPage: React.FC = () => {
+  const siteMetadata = useSiteMetadata();
+
+  const [email, setEmail] = useState('');
+  const [username, setUsername] = useState('');
+  const [password1, setPassword1] = useState('');
+  const [password2, setPassword2] = useState('');
+
+  return (
+    <Layout>
+      <Seo
+        title={`Sign Up`}
+        description={`Sign Up page for ${siteMetadata.title} website`}
+        urlSlug="signup/"
+      />
+      <Wrapper>
+        <Form heading="Sign Up To Join Project Unicorn">
```

Comment:
[2] Hard-coded "Project Unicorn" string...
```suggestion
        <Form heading={`Sign Up To Join ${siteMetadata.title}`}>
```